inc/Engine/Preload/Fonts.php

```diff
@@ -96,18 +99,22 @@ public function preload_fonts() {
 		$fonts = array_filter( $fonts );
 
 		if ( empty( $fonts ) ) {
-			return;
+			return $html;
 		}
 
 		$base_url = get_rocket_parse_url( home_url() );
 		$base_url = "{$base_url['scheme']}://{$base_url['host']}";
 
+		$preloads = '</title>';
+
 		foreach ( array_unique( $fonts ) as $font ) {
-			printf(
+			$preloads .= sprintf(
 				"\n<link rel=\"preload\" as=\"font\" href=\"%s\" crossorigin>",
 				esc_url( $this->cdn->rewrite_url( $base_url . $font ) )
 			);
 		}
+
+		return str_replace( '</title>', $preloads, $html );
```

Comment:
it's better to use `preg_replace()` with a replacement limit of 1 in this case, to prevent inserting in unexpected cases (seen it happen before)